include/taskolib/SequenceManager.h

```diff
@@ -219,6 +225,18 @@ class SequenceManager
      */
     static UniqueId create_unique_id(const std::vector<SequenceOnDisk>& sequences);
 
+    /**
+     * Stage all files git can find in a repository.
+     * parameter dir and fileytpe are used as filter.
+     * To stage all, use dir_name = "" and filetype = ""
+     * filetype values: {"", "new file", "modified", "deleted",
+     * "renamed", "typechange", "untracked"}
```

Comment:
The description of the possible values for filetype should be under the filetype `\param` entry.
Also, does the function work recursively on the contents of the fiven directory? If yes, it should say so. If not, it should explicitly say "non-recursively".

Reply:
My description of the function was not very precise. The dir param is a filter. That means if I only want to stage files from one sequence, I set its directory as dir. By this, I avoid mixing of commits if two users work on two different sequences at once. I added it and I hope it resolves the confusion

Reply:
I still find it somewhat unclear, although by now I think I am starting to understand what it does:

- `Stage all files git can find in a repository.`: Shouldn't this rather be "Make git stage selected changes to files and folders in the base path" or something similar? It certainly does not stage *all* changes except the parameters say so, and I find "in a repository" quite confusing. The function can also stage the *deletion* of files, which is not really included in "stage all files" either.
- `\param dir` is actually called `dir_name`.

It is also still unclear if this stages changes *recursively* or not. In other words, if I add a structure of subfolders like this...
```
sequence1
sequence2
  subfolder_a
  subfolder_b
    subfolder_c
```
... and then call `stage_files_in_directory("sequence2", "")`, does this add also subfolder_c and its contents to the repository?

Reply:
Rewrote `stage_files_in_directory()`; now it uses `git add` and `git update` with pathname globs, so you can specify whatever you like, and of course recurse.